src/App.vue

```diff
@@ -15,6 +15,115 @@
 import { defineComponent } from 'vue';
 
 export default defineComponent({
-    name: 'App'
+    name: 'App',
+
+    computed: {
+        updateAccountSession () {
+            return this.$store.state.account;
+        },
+        updatePlacesSettings () {
+            return this.$store.state.places;
+        },
+        updateAccessToken () {
+            return this.$store.state.account.accessToken;
+        },
+        metaverseServerChanged () {
+            return this.$store.state.metaverseConfig.server;
+        },
+        dashboardConfigStore: {
+            get () {
+                return this.$store.state.dashboardConfig;
+            },
+            set (value) {
+                this.$store.commit('mutate', {
+                    update: true,
+                    property: 'dashboardConfig',
+                    with: value
+                });
+            }
+        },
+        getDashboardTheme () {
+            return this.$store.state.dashboardConfig.dashboardTheme;
+        },
+        isLoggedIn () {
+            return this.$store.state.account.isLoggedIn;
+        }
+    },
+
+    watch: {
+        // Save the state of the session to storage for retrieval if the user leaves and comes back.
+        updateAccountSession: {
+            handler: function (newVal) {
+                for (const item in newVal) {
+                    if (newVal[item] !== null) {
+                        localStorage.setItem(item, newVal[item]);
+                    }
+                }
+            },
+            deep: true
+        },
+        // Save the settings for places.
+        updatePlacesSettings: {
+            handler: function (newVal) {
+                for (const item in newVal) {
+                    if (newVal[item] !== null) {
+                        localStorage.setItem(item, newVal[item]);
+                    }
+                }
+            },
+            deep: true
+        },
+        // Save the settings for the dashboard.
+        dashboardConfigStore: {
+            handler: function (newVal) {
+                for (const item in newVal) {
+                    if (newVal[item] !== null) {
+                        localStorage.setItem(item, newVal[item]);
+                    }
+                }
+            },
+            deep: true
+        },
+        // getDashboardTheme: {
+        //     handler: function () {
+        //         this.setDashboardTheme();
+        //     }
+        // },
+        updateAccessToken: {
+            handler: function () {
+                console.info('Setting new access token header...');
+                window.$.ajaxSetup({
+                    beforeSend: (xhr) => {
+                        xhr.setRequestHeader('x-vircadia-error-handle', 'badrequest');
+                        xhr.setRequestHeader('Authorization', 'Bearer ' + this.$store.state.account.accessToken);
+                    }
+                });
+            }
+        },
+        metaverseServerChanged: {
```

Comment:
if you're not doing a deep watch, would this be simpler to use the "shortcut version" of this and have metaverseServerChanged just be a function?